### PR TITLE
Remove metric options from add command for dataset and labels

### PR DESCRIPTION
**Pull request by adysonl**
The metrics option is only supported for models but displayed when inserting the command "ml-git datasets add --help" or "ml-git labels add --help".

### CODE REVIEW

**Review by lucasncabral — APPROVED** [review]

File: ml_git/commands/entity.py [hunk 1]
```diff
@@ -119,8 +119,8 @@ def add(context, **kwargs):
     run_fsck = kwargs['fsck']
     file_path = kwargs['file_path']
     entity_name = kwargs['ml_entity_name']
-    metric = kwargs['metric']
-    metrics_file_path = kwargs['metrics_file']
+    metric = kwargs.get('metric')
```
Comment (gilMars) [inline ml_git/commands/entity.py:122]: What's the difference between get method and use the key directly?

File: tests/integration/test_29_log.py [hunk 2]
```diff
@@ -112,17 +116,3 @@ def test_05_log_metrics(self):
         self.assertNotIn(output_messages['INFO_ADDED_FILES'], output)
         self.assertNotIn(output_messages['INFO_DELETED_FILES'], output)
         self.assertIn(self.create_metrics_table(), output)
-
-    @pytest.mark.usefixtures('start_local_git_server', 'switch_to_tmp_dir')
-    def test_06_log_metrics_wrong_entity(self):
```
Comment (gilMars) [inline tests/integration/test_29_log.py:117]: Why did you remove this test?

File: ml_git/commands/entity.py (hunk 1 above)
Comment (adysonl) [reply ml_git/commands/entity.py:122]: Using the key directly would throw a KeyError if metric is not in the kwargs dict. Using get will just set the variable as None.

File: tests/integration/test_29_log.py (hunk 2 above)
Comment (adysonl) [reply tests/integration/test_29_log.py:117]: Since the add command no longer work using metric options, the test would break in the setup (line 31). As it is testing if the metrics are not being logged for labels and datasets I added the `self.assertNotIn(self.create_metrics_table(), output)` to the other existing tests.

**Review by gilMars — APPROVED** [review]

**Review by andersonpablosilva — APPROVED** [review]
